### PR TITLE
add def comp info for pytket-qiskit

### PR DESCRIPTION
**Pull request by CalMacCQ**
I've added a list of backends to the docs page for pytket qiskit. I've also added info on the default_compilation_pass for different levels of optimisation. 

Let me know if I've missed anything and if more detail is needed.

### CODE REVIEW

File: modules/pytket-qiskit/docs/intro.txt [hunk 1]
```diff
@@ -16,7 +16,70 @@ representations.
...
+   * -
+     - self.rebase_pass [3]
+     - `RemoveRedundancies <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.RemoveRedundancies>`_
+   * - 
+     -
+     - `SimplifyInitial <https://cqcl.github.io/tket/pytket/api/passes.html#pytket.passes.SimplifyInitial>`_
+     
+* [1] If no value is specified then ``optimisation_level`` defaults to a value of 1.
+* [2] Here ``CXMappingPass`` maps program qubits to the architecture using a `NoiseAwarePlacement <https://cqcl.github.io/tket/pytket/api/placement.html#pytket.placement.NoiseAwarePlacement>`_
+* [3] self.rebase_pass is a rebase to the IBM native gate set (e.g. {X, SX, Rz, CX}).
```
Comment (ss2165) [inline modules/pytket-qiskit/docs/intro.txt:81]: "gate set supported by the backend, for IBM quantum devices that is {X, SX, Rz, CX}"

File: modules/pytket-qiskit/docs/intro.txt [hunk 2]
```diff
@@ -16,7 +16,70 @@ representations.
...
+   * - `IBMQEmulatorBackend <https://cqcl.github.io/pytket-extensions/api/qiskit/api.html#pytket.extensions.qiskit.IBMQEmulatorBackend>`_
+     - Emulator for a chosen ``IBMBackend`` (Device specific).
+   * - `AerBackend <https://cqcl.github.io/pytket-extensions/api/qiskit/api.html#pytket.extensions.qiskit.AerBackend>`_
+     - A noiseless, shots-based simulator for quantum circuits [1]
+   * - `AerStateBackend <https://cqcl.github.io/pytket-extensions/api/qiskit/api.html#pytket.extensions.qiskit.AerStateBackend>`_
+     - Statevector simulator.
+   * - `AerUnitaryBackend <https://cqcl.github.io/pytket-extensions/api/qiskit/api.html#pytket.extensions.qiskit.AerUnitaryBackend>`_ 
+     - Unitary simulator
+
+* [1] By default ``AerBackend`` is noiseless by default and has no architecture. However it can accept a user defined ``NoiseModel`` and ``Architecture``.
```
Comment (ss2165) [inline modules/pytket-qiskit/docs/intro.txt:43]: Worth adding more detail on how to do noisy simulation

**Review by ss2165 — CHANGES_REQUESTED** [review]
Things I spotted on the quantinuum intro worth including here https://github.com/CQCL/pytket-quantinuum/blob/develop/docs/intro.txt:

* predicates
* statuses
* backend init parameters, most importantly how to authenticate with a particular hub/group/use a specific authentication provider.
* how batching works
* kwargs to process_circuits and get results
* what the result handle means
*

**Comment by cqc-melf** [comment]
Is this something you want to get moved over to the new repo? https://github.com/CQCL/pytket-qiskit/

**Comment by CalMacCQ** [comment]
Yes, if thats possible to do. 

This was something I meant to finish ages ago but moved onto other things. I think it would be worth finishing this soon.

I also have a copy of everything locally

**Comment by cqc-melf** [comment]
Okay, will do that.

**Comment by cqc-melf** [comment]
reopened at https://github.com/CQCL/pytket-qiskit/pull/3

**Pull request closed without merging**